test/uk/gov/hmrc/gform/core/parsers/ValueParserSpec.scala

```diff
@@ -179,7 +179,7 @@ class ValueParserSpec extends FlatSpec with Matchers with EitherValues with Opti
     )
   }
 
-  val plainFormTemplate = FormTemplate(FormTypeId("IPT100"), "Insurance Premium Tax Return", "version", "description", "characterSet", DmsSubmission("nino", "BT-NRU-Environmental", "FinanceOpsCorpT"), "submitSuccessUrl", "submitErrorUrl", List.empty[Section])
+  val plainFormTemplate = FormTemplate(Some("schemaId"), FormTypeId("IPT100"), "Insurance Premium Tax Return", "version", "description", "characterSet", DmsSubmission("nino", "BT-NRU-Environmental", "FinanceOpsCorpT"), "submitSuccessUrl", "submitErrorUrl", List.empty[Section])
```

Comment:
There is a nicer cats syntax available after importing `import cats.syntax.all._`. Consider:
`"http:\\schemaId".some` vs `Some("http:\\schemaId")`